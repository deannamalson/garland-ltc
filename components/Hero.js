import React from "react";
import Links from "./global/Links";

const Hero = () => (
  <div>
    <h1>GARLAND LTC</h1>
    <Links />
  </div>
);

export default Hero;
