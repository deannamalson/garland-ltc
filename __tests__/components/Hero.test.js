import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import Hero from "../../components/Hero";

test("should render Hero correctly", () => {
  const wrapper = shallow(<Hero />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
