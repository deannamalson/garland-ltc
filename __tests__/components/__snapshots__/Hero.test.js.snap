// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Hero correctly 1`] = `
<div>
  <h1>
    GARLAND LTC
  </h1>
  <Links />
</div>
`;
